Update of README (#4)
OSM-Catalan/LangToolsOSM: README.md

You will be asked for necessary options if they are not passed to the command call (```--area```, ```--lang```, ```--username```).

You can define the search area by the coordinates of the bounding box in the following format ```(South,West,North,East)```, overpass filters or by the exact ```name``` value of a feature with area.

# Example

> In the following example you will be able to add all the ```name:ca``` that are missing in the municipality of Alcalalí with the name of the ```name```.

Reviewer: ```suggestion
In the following example you will be able to review and add all the ```name:ca``` that are missing in the municipality of Alcalalí with the content of the ```name``` tag which is in catalan.
```

Author: He posat windows perqué jo utilitze windows, encara que si les instruccions són iguals per als dos O.S. es pot eliminar perfectament la menció a windows
